src/escalus_session.erl

```diff
@@ -121,9 +121,9 @@ session(Conn, Props) ->
     Props.
 
 use_ssl(Props, Features) ->
-    UserNeedSSL = proplists:get_value(ssl, Props, false),
-    StreamAllowSSL = proplists:get_value(starttls, Features),
-    case {UserNeedSSL, StreamAllowSSL} of
+    UserNeedsSSL = proplists:get_value(starttls, Props, false),
```

Comment:
It was reflected in tests - test/test.config. And the PR was not merged immediately but only one person bothered to review it (before merge :).